includes/html/pages/edituser.inc.php

```diff
@@ -112,7 +124,65 @@
            </div>
            <button type='submit' class='btn btn-default' name='Submit'>Add</button></form>";
 
-        echo "</div>
+        echo '</div>
+           <div class="col-md-4">';
+
+        // Display devices this users has access to
+        echo '<h3>Device Group Access</h3>';
+
+        echo "<div class='panel panel-default panel-condensed'>
+            <table class='table table-hover table-condensed table-striped'>
+              <tr>
+                <th>Device Group</th>
+                <th>Action</th>
+              </tr>";
+
+        $devices_group_perms = dbFetchRows('SELECT DGP.device_group_id, DP.name from devices_group_perms DGP INNER JOIN device_groups DP ON DP.id = DGP.device_group_id  WHERE DGP.user_id = ?', array($user_data['user_id']));
```

Comment:
You don't have to use the db functions in legacy code...  Eloquent is available.

Reply:
That entire page needs to be rewritten anyway